<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitForSoftimage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="banner_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 bold italic 18pt &quot;Trebuchet MS&quot;;
color: rgb(220, 220, 220);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 20, 20, 255), stop:1 rgba(40, 40, 40, 255));</string>
     </property>
     <property name="text">
      <string>GitForSoftimage</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="margin">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contents_hlayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="data_vlayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="options_frame">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="reload_button">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="branch_button">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Branch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>master</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="options_hspacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="history_button">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;History&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>History</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remotes_button">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remotes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Remotes</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="terminal_button">
            <property name="text">
             <string>Terminal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="prefs_button">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Preferences&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Prefs</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="files_listWidget"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="commit_vlayout">
       <item>
        <widget class="QLineEdit" name="message_lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Commit Message (required)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="commit_hlayout">
         <item>
          <spacer name="commit_hspacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="commit_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Commit</string>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="commit_vspacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>message_lineEdit</tabstop>
  <tabstop>commit_button</tabstop>
  <tabstop>reload_button</tabstop>
  <tabstop>branch_button</tabstop>
  <tabstop>history_button</tabstop>
  <tabstop>remotes_button</tabstop>
  <tabstop>terminal_button</tabstop>
  <tabstop>prefs_button</tabstop>
  <tabstop>files_listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>files_listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>file_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commit_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>commit_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>history_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>history_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prefs_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>prefs_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branch_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>branch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reload_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reload_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remotes_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remotes_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terminal_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>terminal_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>file_clicked()</slot>
  <slot>commit_clicked()</slot>
  <slot>history_clicked()</slot>
  <slot>prefs_clicked()</slot>
  <slot>branch_clicked()</slot>
  <slot>reload_clicked()</slot>
  <slot>remotes_clicked()</slot>
  <slot>terminal_clicked()</slot>
 </slots>
</ui>
